datafusion/sqllogictest/test_files/order.slt

```diff
@@ -888,6 +888,30 @@ select column2, column1 from foo ORDER BY column2 desc, column1 desc;
 [1] 4
 [0] 2
 
+# Test issue: https://github.com/apache/datafusion/issues/10013
+# There is both a HAVING clause and an ORDER BY clause in the query.
+query I
+SELECT
+   SUM(column1)
+FROM foo
```

Comment:
Could we also add an example that doesn't have a HAVING as well as one with GROUP BY? something like this perhaps

```sql
SELECT SUM(column1) FROM foo ORDER BY SUM(column1)
```

```sql
SELECT column2 FROM foo ORDER BY SUM(column1)
```

```sql
SELECT SUM(column1) FROM foo ORDER BY SUM(column1) GROUP BY column2
```

Reply:
Added. Thank you @alamb !